src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
```

Comment:
Checklist p3 -  [NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)